Modules/Module_Import/Import/xASL_imp_DetermineStructureFromSourcedata.m

```diff
@@ -289,24 +303,15 @@ function xASL_imp_PrintRun(x,thisSubject,thisVisit,thisRun)
 				end
 
 				% Resolve the new name of the visit
-				nameVisit = x.modules.import.imPar.tokenVisitAliases{idVisit, 2};
-				
-				% Extract the number of the visit
-				numberVisit = regexp(nameVisit, '\d+');
-				if ~isempty(numberVisit)
-					numberVisit = nameVisit(numberVisit:end);
-				else
-					numberVisit = sprintf('%g', idVisit);
-				end
-				x.modules.import.imPar.visitNames{iVisit} = sprintf('ASL_%s', numberVisit);
+				x.modules.import.imPar.visitNames{iVisit} = x.modules.import.imPar.tokenVisitAliases{idVisit, 2};
```

Comment:
What happened here? What you write looks good but I don't know if there is a reason (== use case) for the `sprintf('ASL_%s', numberVisit` and other stuff you just deleted

Reply:
Very simple - previously, we only used numbers for visit names and we extracted them as numerical. Now, we save everything as string, so if we have a number in a string, we just save it without doing the number->string conversion.

Reply:
1.No need. xASL_num2str already does this for us.

xASL_num2str('20240203') == '20240203'
xASL_num2str(20240203) == '20240203'

So can we simplify our code with this?

2.Something else, why did we have 'ASL_%s' if this was about visits? You're sure your not removing an 'ASL_' prefix that should have stayed here?

Reply:
1. True, I was simplifying, but didn't realize I could simplify more. Done in the last commit.

2. No clue why it was there - ask Michael. I'm pretty sure this was a nonsense and the whole Flavor import works, so that confirms my suspicion.